<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharServer</class>
 <widget class="QMainWindow" name="CharServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CharServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入端口号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>启动服务器</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>411</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>111</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
